process/templates/SIG_NAME_graduation_stage.md

```diff
@@ -1,5 +1,31 @@
 ## Special Interest Group (SIG) graduation
 
 The governing body must agree that the SIG has completed a major deliverable
-
   * Link to relevant documentation.
+
+## SIG has met all Incubating requirements
+  * "link to Incubating PR"
```

Comment:
Sorry but this is a pain for people to go dig up and we should have the corresponding file in the `process/sig-lifecycle-documents` folder.
Plus doesn't that mean we are forcing every SIG to go through every step then?
We agreed that given that we have already had a lot of TIs in existence for a long time now we will allow TIs to apply for whatever stage they think they qualify for. This means that there may not be an "Incubating PR".

Reply:
I have no worries grandfathering in existing TIs, but I feel this is a useful aspirational thing we should keep for all future work.  I also agree that we need somewhere to store all of these completed ones.  What would a good path be?  A folder for each TI type?  One giant folder for all TIs?  A folder per lifecycle stage?

Reply:
We already have these folders in place and the instructions on how to use them:

- https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#working-group-creation-or-change-of-lifecycle-stage
- https://github.com/ossf/tac/blob/main/process/project-lifecycle.md#project-creation-or-change-of-lifecycle-stage
- https://github.com/ossf/tac/blob/main/process/sig-lifecycle.md#sig-creation-or-change-of-lifecycle-stage

If this is followed, the questionnaire in which this question is being asked will end up right next to the one that was filed for the move to Incubation, along with every other stage change.

In the case of a SIG we will end up with something like:

process/sig-lifecycle-documents/MyFavoriteSIG-sandbox-stage.md
process/sig-lifecycle-documents/MyFavoriteSIG-incubation-stage.md
process/sig-lifecycle-documents/MyFavoriteSIG-graduation-stage.md
process/sig-lifecycle-documents/MyFavoriteSIG-archived-stage.md

Giving us direct access to the whole history in one location.

Reply:
Have the adjustments I've made helped clarify this to resolve and move forward?